transactions/transactions.py

```diff
@@ -273,6 +282,14 @@ def get_player_nickname(self, user : discord.Member):
             return currentNickname
         return user.name
 
+    async def get_tier_role_for_fa(self, ctx, user : discord.Member):
```

Comment:
The user is being passed into this function, but not being used. I assume you want to filter the list of FA roles somehow using the user's current roles?

Reply:
Yes, that's generally what I intended. My mistake, I thought I wrapped this up, but I guess I didn't. But yes, filtering the FA roles from the user was and is plan. According to my tests, this works, but I will clean it up before we complete the PR.

Reply:
After I did some thinking, it probably worked simply because I was only testing against 2 tiers. I'll be sure to fix this function.

Reply:
This should be resolved now.